src/codeu/chat/client/simplegui/CreateAccountPanel.java

```diff
@@ -1,42 +1,47 @@
 package codeu.chat.client.simplegui;
 
 import javax.swing.*;
+
+import codeu.chat.client.simplegui.NewAccount;
+
 import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
 
 /**
- * Created by Suveena on 5/18/17.
+ * @author  Suveena
+ * @date    5/18/17
+ * @brief   This class contains a button for the user to create an account.
+ *          This panel contains from top to bottom; a message, a button.
  */
 
 @SuppressWarnings("serial")
-public class CreateAccountPanel extends JPanel{
+public class CreateAccountPanel extends JPanel {
 
-    public CreateAccountPanel() {
+    CreateAccountPanel() {
```

Comment:
why did you remove public? did you not want any "outsiders" to create a CreateAccountPanel?